MyBus/MyBus/Model/Road/RoutePoint.swift

```diff
@@ -34,6 +34,34 @@ class RoutePoint: NSObject {
         }
     }
 
+    static func parseFromGeoGoogle(geoPointJson: JSON) -> RoutePoint? {
+        let geoPoint = RoutePoint()
+        let firstResultJson = geoPointJson["results"][0]
+        let isAddress = firstResultJson["address_components"][0]["types"] == [ "street_number" ]
+        let jsonStatus = geoPointJson["status"].stringValue
+
+        guard isAddress else {
+            return nil
+        }
+
+        switch jsonStatus {
+        case "OK":
```

Comment:
Perhaps could be a good idea to store the "OK" string in a static variable or in a enumeration so if in the future the json replies differently could be changed easily